dom-events/document.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListener('DOMContentLoaded', function (event) {
+  console.log('document loaded');
+});
+
+document.addEventListener('keydown', function (event) {
+  console.log('key pressed');
+  console.log('  event.key:', event.key);
+});
+
+document.addEventListener('mousemove', function (event) {
+  console.log('mouse moved');
+  console.log('  pageX:', event.pageX, 'pageY:', event.pageY);
+});
```

Comment:
Please use named functions and callbacks rather than passing the full function definition as an argument